### PR TITLE
Add org.kde.alligator

### PR DESCRIPTION
**Pull request by ognarb**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] @TobiasFella agreed to publish Alligator on Flathub
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned.

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by hfiguiere** [comment]
bot, build org.kde.alligator

File: org.kde.alligator.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    "sdk": "org.kde.Sdk",
+    "command": "alligator",
+    "finish-args": [
+        "--share=ipc",
+        "--share=network",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--device=dri"
+    ],
+    "separate-locales": false,
```
Comment (bilelmoussaoui) [inline org.kde.alligator.json:15]:
```suggestion
    "separate-locales": false,
```
is this really needed?

**Comment by nedrichards** [comment]
@flathub/kde FYI

**Comment by ognarb** [comment]
bot, build org.kde.alligator

**Comment by travier** [comment]
Not test yet but LGTM

**Pull request closed without merging**